libs/libmstch/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mstch
+PKG_VERSION:=1.0.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/no1msd/mstch/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=811ed61400d4e9d4f9ae0f7679a2ffd590f0b3c06b16f2798e1f89ab917cba6c
+
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILE:=LICENSE
+
+PKG_BUILD_DEPENDS:=boost
+PKG_CONFIG_DEPENDS:=CONFIG_PACKAGE_boost-container
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+PKG_BUILD_PARALLEL:=1
+CMAKE_INSTALL:=1
+EXTRA_CXXFLAGS:=$(FPIC)
```

Comment:
Use TARGET_CFLAGS instead?

Reply:
include/package-defaults.mk shows
```
CXXFLAGS="$(TARGET_CXXFLAGS) $(EXTRA_CXXFLAGS)"
```
moot point. Anyway. Two things I don't like here. CMake itself has CMAKE_POSITION_INDEPENDENT_CODE that can be used.

I also remember a C++ package recently I messed with where adding FPIC to TARGET_CXXFLAGS did not fix compilation but did when added under TARGET_CFLAGS.

I think it's better to let CMake handle it here.

Reply:
I'll try -DCMAKE_POSITION_INDEPENDENT_CODE=True and see if that works instead.